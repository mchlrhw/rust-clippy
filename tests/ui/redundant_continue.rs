#![warn(clippy::redundant_continue)]

fn main() {
    let x = 42;

    loop {
        if x == 0 {
            println!("foo");
        }
    }

    loop {
        //~ ERROR trailing continue
        continue;
    }

    loop {
        if x == 0 {
            println!("foo");
        }
        //~ ERROR trailing continue
        continue;
    }
}
