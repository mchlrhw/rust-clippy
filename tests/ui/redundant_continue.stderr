error: loop with trailing `continue`
  --> $DIR/redundant_continue.rs:14:9
   |
LL |         continue
   |         ^^^^^^^^
   |
   = note: `-D clippy::redundant-continue` implied by `-D warnings`
   = help: consider removing

error: loop with trailing `continue`
  --> $DIR/redundant_continue.rs:22:9
   |
LL |         continue
   |         ^^^^^^^^
   |
   = help: consider removing

error: aborting due to 2 previous errors

